### ISSUE TITLE
To create wallet

### ISSUE THREAD

**Issue by spArTans5** [Ed-Yang/xilinx-ethash#5]
How do you create wallet for this particular project

**Comment by Ed-Yang** [comment]
I do not understand your question, please elaborate, thanks.

**Comment by spArTans5** [comment]
We need to add our wallet address before mining. Which is the particular directory or is there procedure for that 
Thank you

**Comment by Ed-Yang** [comment]
The original plan is to port Ethminer to Xilinx U50 card with two steps.  The first step is measure performance od ethash on U50, and then integrate it to Ethminer.  In the first step, it is found the performance is too slow (see README), so I release the work to GitHub for getting help.  If there is a way to improve ethash performance, it is easy to make it work on Ethminer.

This project does not provide a full miner program, it is just a demo of ethash on Xilinx U50, so it does not need to provide the wallet address.

**Comment by spArTans5** [comment]
Thank You ,do you have any idea how that can be done

**Comment by spArTans5** [comment]
Is there the need for vitis to bulid ethash?if yes,can I know the procedure
Thank You

**Comment by Ed-Yang** [comment]
Yes, it is needed to install Vitis to build this project.  The ethash implementation is copied from Ethminer project which is written in OpenCL.

**Comment by spArTans5** [comment]
ethash is the only thing that needs to be build in vitis?

**Comment by spArTans5** [comment]
Do you have any how on to increase the performance in alveo U50 or is it a dead end ?
Thank You

**Comment by Ed-Yang** [comment]
I am not expert at Xilinx programming, the following link is my post for performance issue.  It is believed
that are some way to improve it.

https://forums.xilinx.com/t5/Vitis-Acceleration-SDAccel-SDSoC/Run-ethash-OpenCL-kernel-on-Alveo-U50-PCIe-card-is-much-slower/m-p/1209155

**Comment by spArTans5** [comment]
Thank You
